README.md

```diff
@@ -1,26 +1,48 @@
-# Role Name
+# Limesurvey
 
-A brief description of the role goes here.
-
-
-## Requirements
-
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
+A role to set up and configure a limesurvey instance.
 
```

Comment:
What are the requirements for this role?

Reply:
done